### ISSUE TITLE
UI improvments...

### ISSUE THREAD

**Issue by ng8weji9vowdv** [mifi/lossless-cut#2116]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

.

**Comment by ng8weji9vowdv** [comment]
So, the main and primary issue for my case is when i am using the scene detector and let's say it creates 10000 segments, and the tool fails to process them after 8-9%, and stucks there with no progress being applied neither to the files nor in UI.

The other issue, is obvious being unable to put segments back together, unite them, instead of splitting (which makes the tool pretty lame, its just an array of segments, why can't we just unite 2 or more segments and work with them as a unit?)...

The third issue, this annoying ass bar that overlaps the segments.
![image](https://github.com/user-attachments/assets/2beefe06-64d0-4a60-a3eb-aec8d9710933)
Like imagine being on the edge of a segment and this thing overlaps it? Lol. 

Besides that, these zoom in zoom out animates are horrendous but no options to turn them off truly, it wouldve been much better if while zooming in id not have to wait 3 seconds until the things appear under my mouse cursor.

Besides that animation issue the zooming itself has an individual issue. let's say you are zooming in and you're on like 4096x zoom, and you use ctrl + mouse wheel down you expect it step down, but no, instead it goes on zoom 1x lol.

Also, the segments bar, as i said let's imagine we have like 10000 segments, and there's no scroll bar you literally used to either seek in the bar or scroll down.

To implement the segments join (unite) you have to also add support of selecting items with ctrl + click, and the other way is:
select item
scroll down all until you meet the needed segment and use ctrl + shift or something to select all segment between two segments.

It can work for both individual segments selection (as in final result) or to join them later via the context menu.

And finally, since you're creating the llc file `-proj.llc`, then why do we not have a simple export all segments into a json file button? Additionally it would be nice to have a simple seconds (not milliseconds) counter in the progress bar or at least see the seconds/timestamp (should be available to switch) of segment start and segment end in segments list.
![image](https://github.com/user-attachments/assets/a5d982a7-f025-43fc-8071-43818e6a27f4)

Because here in segments json file we only have the areas in seconds (right now we only can observe start end in regular time alike and the duration is in ms).



Overall, tool is fine.

**Comment by mifi** [comment]
> So, the main and primary issue for my case is when i am using the scene detector and let's say it creates 10000 segments, and the tool fails to process them after 8-9%, and stucks there with no progress being applied neither to the files nor in UI.

I think #1727

> I ended up manually creating segments and idk, rendering them ig? Annoying a little, so id rather have it done automatically in the correct way.

It's hard to follow your explanations. maybe you can create a bug report with exact steps to reproduce the problem and what you see and what you expect t osee:

https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&projects=&template=2-bug.yml

> Like imagine being on the edge of a segment and this thing overlaps it, because of centering (oh yeah, sometimes it goes outside of the observable space, so the cursor gets off the screen, somewhere in the right or left depending where you are at)? Lol.

your screenshot does not illustrate what's the problem. why is it a problem that the translucent background overlaps a segment?

> oh yeah, sometimes it goes outside of the observable space

I don't know what that means

> Besides that, these zoom in zoom out animations (and the rest of them too, select unselect segment for example) are horrendous but no options to turn them off truly, it wouldve been much better if while zooming in id not have to wait 3 seconds until the things appear under my mouse cursor.

you can enable "reduced motion" in your OS and it should respect that setting and disable animations in the app. does that work?

> let's say you are zooming in and you're on like 4096x zoom, and you use ctrl + mouse wheel down you expect it step down, but no, instead it goes on zoom 1x lol.

not able to repro this. when i ctrl+wheel down it doesn't go to 1x. could you record a video or screenshots with explanation?

> and there's no scroll bar you literally used to either seek in the bar or scroll down.

I will add a scrollbar

> And finally, since you're creating the llc file `-proj.llc`, then why do we not have a simple export all segments into a json file button, for later processing via ffmpeg manually.

not sure what you mean by this but File -> export project(llc) will export all segments into a json file.

> Additionally it would be nice to have a simple seconds (not milliseconds) counter in the progress bar or at least see the seconds/timestamp (should be available to switch) of segment start and segment end in segments list.

which progress bar are you talking about that has a milliseconds counter?

> Because here in segments json file we only have the areas in seconds (right now we only can observe start end in regular time alike and the duration is in ms).

?

> it should display all of them before running them, just add into the list and let people run manually.

not clear how this should work. #974 ?

**Comment by mifi** [comment]
closing due to lack of response
